webui/react/src/components/kit/Button.module.scss

```diff
@@ -0,0 +1,3 @@
+.wrapped {
+  margin-left: 8px;
```

Comment:
question: What is this doing?

Reply:
antd applies a margin to the submit button, which doesn't get applied when there's a wrapping element around the button, so the intent here was to make sure that spacing is preserved. but looking at it again after you asked about it, I think this is actually done by the antd styles for the modal, so i'll move it there. thanks!